lib/deps/docs/processDocs.js

```diff
@@ -15,6 +15,10 @@ function processDocs(revLimit, docInfos, api, fetchedDocs, tx, results,
   // Default to 1000 locally
   revLimit = revLimit || 1000;
 
+  function rootIsMissing(docInfo) {
+    return docInfo.metadata.rev_tree[0].ids[1].status === "missing";
+  }
+
```

Comment:
this would be better for performance to hoist to the outer function, since it doesn't need any local variables